app/src/main/java/com/framgia/fdms/utils/binding/BindingUtils.java

```diff
@@ -452,10 +444,10 @@ public static void setImageBitmap(ImageView view, Bitmap bitmap) {
 
     @BindingAdapter("setVisibility")
     public static void setVisibility(com.github.clans.fab.FloatingActionButton view,
-            RequestDetailViewModel viewModel) {
+                                     RequestDetailViewModel viewModel) {
         int visibility = viewModel.getStatusRequest().equals(Constant.DeviceStatus.DONE)
-                || viewModel.getStatusRequest().equals(Constant.DeviceStatus.APPROVED) ? View.GONE
-                : View.VISIBLE;
+            || viewModel.getStatusRequest().equals(Constant.DeviceStatus.APPROVED) ? View.GONE
```

Comment:
WARNING: [UnnecessaryFullyQualifiedName] Unnecessary use of fully qualified name 'Constant.DeviceStatus.APPROVED' due to existing static import 'com.framgia.fdms.utils.Constant.DeviceStatus.APPROVED'
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#UnnecessaryFullyQualifiedName